packages/vue-instantsearch/src/components/Hits.js

```diff
@@ -1,5 +1,6 @@
 import { createHitsComponent } from 'instantsearch-ui-components';
 import { connectHitsWithInsights } from 'instantsearch.js/es/connectors';
+import { Fragment } from 'preact';
```

Comment:
This shouldn't be imported as it adds around 3kb to the bundle

Reply:
I noticed the increase, thank you for mentioning this and for the code snippet. I'll push another commit to update `bundlesize` when it goes back down after removing.